src/components/Table/utils/RowState.ts

```diff
@@ -59,6 +67,7 @@ export class RowState<R extends Kinded> {
         _row: false,
         _data: observable.ref,
         isCalculatingDirectMatch: false,
+        isDraggedOver: observable.ref, // allows the table to re-render only this row when the dragged over state changes
```

Comment:
Question: Given that `isDraggedOver` is a primitive, I _think_ this is unnecessary? Usually the `observable.ref` short-circuits mobx from doing a deep compare, which is how primitives basically work by default.